openshift/e2e-common.sh

```diff
@@ -148,10 +148,11 @@ spec:
         "{{.Revision.PodIP}}", "referer": "{{js .Request.Referer}}", "latency": "{{.Response.Latency}}s",
         "protocol": "{{.Request.Proto}}"}, "traceId": "{{index .Request.Header "X-B3-Traceid"}}"}'
       logging.enable-probe-request-log: "true"
+      logging.enable-request-log: "true"
```

Comment:
Upstream added this by https://github.com/knative/serving/commit/f7462cd8a4cfed3f3b9d7fb261c8f20a3e45deb7#diff-9eeb566948e735def9872b88b198199aR27